controllers/scripts/config.go

```diff
@@ -236,7 +238,10 @@ func applyIPFSClusterK8sDefaults(conf *config.Config, storageMax string, peers [
 	conf.Bootstrap = config.DefaultBootstrapAddresses
 	conf.Addresses.API = config.Strings{"/ip4/0.0.0.0/tcp/5001"}
 	conf.Addresses.Gateway = config.Strings{"/ip4/0.0.0.0/tcp/8080"}
-	conf.Swarm.ConnMgr.HighWater = 2000
+	highWater := &config.OptionalInteger{}
+	if err := highWater.UnmarshalJSON([]byte("2000")); err == nil {
+		conf.Swarm.ConnMgr.HighWater = highWater
```

Comment:
?

Reply:
In the upgraded version of kubo `conf.Swarm.ConnMgr.HighWater` is expecting `*config.OptionalInteger`. 
I was getting the below error hence I changed it. 
`cannot use 2000 (untyped int constant) as *config.OptionalInteger value in assignment`

Reply:
Oh okay, I see that now. Thanks!